bigcache_test.go

```diff
@@ -27,6 +27,23 @@ func TestWriteAndGetOnCache(t *testing.T) {
 	assertEqual(t, value, cachedValue)
 }
 
+func TestAppendAndGetOnCache(t *testing.T) {
+	t.Parallel()
+
+	// given
+	cache, _ := NewBigCache(DefaultConfig(5 * time.Second))
+	value := []byte("value")
+
+	// when
+	cache.Append("key", value)
+	cache.Append("key", value)
```

Comment:
We need a test to append different value and to run multiple appends in parallel.

Reply:
improved the test, the test is already marked with t.Parallel() is that what you mean?

Reply:
I thought about doing sometehing like we did here https://github.com/allegro/bigcache/blob/master/bigcache_test.go#L405-L406 just to check if locking is done properly.

Reply:
ok good call, I implemented something like you mentioned and found another bug :)

One problem remains: the race detector triggers but I have no idea why. Do you guys have experience with it? Is it "valid" all the time?

Reply:
Ok fixed it, it was because I was using the "err" variable from the outer scope! :)